python/src/scipp/neutron/instrument_view.py

```diff
@@ -31,6 +31,7 @@ def instrument_view(scipp_obj=None,
                     continuous_update=True,
                     dim="tof",
                     rendering="Full",
+                    pixel_size=0.0005,
```

Comment:
Maybe a future upgrade, but would it be better if we attempt to calculate this based on detector offsets rather than hard-code it? My main concern is where the pixel_size ends up too large and overlaps neighbours.

Reply:
That is why it is an option now. Previously it was hard-coded. I think we should rather focus on making the full geometry loading faster, so people don't have to resort to this hack.